src/App.js

```diff
@@ -2,14 +2,17 @@ import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
 import "./App.css";
 import Login from "./pages/login";
 import Home from "./pages/home";
+import Card from "./pages/card";
 
 function App() {
   return (
     <div className="App">
+      <Card />
```

Comment:
`Card` is a re-usable component and not a page. We will use this component inside the page `Home` eventually. I think we can remove this here.